nncf/openvino/graph/metatypes/openvino_metatypes.py

```diff
@@ -667,6 +667,8 @@ class OVAbsMetatype(OVOpMetatype):
 # Contains the operation metatypes for which bias can be applied.
 OPERATIONS_WITH_BIAS_METATYPES = [
     OVConvolutionMetatype,
+    OVGroupConvolutionMetatype,
+    OVDepthwiseConvolutionMetatype,
```

Comment:
These changes may affect the BC and FBC algorithms.

Reply:
I reverted this changes as new approach of automatic biases insertion won't need to call `node_with_biases` function